src/stan/lang/function_signatures.h

```diff
@@ -785,8 +785,19 @@ for (size_t i=1; i < 10; i++) {
   add("num_elements", expr_type(int_type()), expr_type(vector_type(), i));
 }
 add("ordered_logistic_log", expr_type(double_type()), expr_type(int_type()), expr_type(double_type()), expr_type(vector_type()));
+add("ordered_logistic_log", expr_type(double_type()), expr_type(int_type(), 1), expr_type(vector_type()), expr_type(vector_type()));
```

Comment:
I'm not sure what happened to cpplint---it must not be checking this, but we should still try to keep to 80 characters max.  This looks longer, but I can't quite tell here.